provider/datagen/src/bin/datagen.rs

```diff
@@ -2,17 +2,24 @@
 // called LICENSE at the top level of the ICU4X source tree
 // (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
 
-use clap::{App, Arg, ArgGroup};
+use clap::{crate_authors, crate_version, App, Arg, ArgGroup};
 use eyre::WrapErr;
 use icu_datagen::prelude::*;
 use simple_logger::SimpleLogger;
 use std::path::PathBuf;
 
 fn main() -> eyre::Result<()> {
-    let matches = App::new("ICU4X Data Exporter")
-        .version("0.0.1")
-        .author("The ICU4X Project Developers")
-        .about("Export CLDR JSON into the ICU4X data schema")
+    let matches = App::new("icu4x-datagen")
+        .version(crate_version!())
+        .author(crate_authors!())
+        .about(concat!(
```

Comment:
Suggestion: this should link to docs.rs and docs.rs should have a link to the tutorial. Alternatively, just always link to the latest tutorial.

Reply:
done